l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py

```diff
@@ -518,6 +518,29 @@ def test_23_xml_import(self):
         invoices = self.invoice_model.browse(invoice_ids)
         self.assertEqual(len(invoices), 2)
 
+    def test_24_xml_import(self):
+        arrotontamenti_attivi_account_id = self.env['account.account'].\
```

Comment:
```suggestion
        arrotondamenti_attivi_account_id = self.env['account.account'].\
```
qui e sotto, piccolo typo